<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'validator.not_compromised_password' shared service.

include_once \dirname(__DIR__, 7).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
include_once \dirname(__DIR__, 7).'/vendor/symfony/validator/ConstraintValidator.php';
include_once \dirname(__DIR__, 7).'/vendor/symfony/validator/Constraints/NotCompromisedPasswordValidator.php';
include_once \dirname(__DIR__, 7).'/vendor/symfony/http-client-contracts/HttpClientInterface.php';
include_once \dirname(__DIR__, 7).'/vendor/symfony/http-client/HttpClient.php';

$a = \Symfony\Component\HttpClient\HttpClient::create([], 6);
$a->setLogger(($this->privates['logger'] ?? ($this->privates['logger'] = new \Symfony\Component\HttpKernel\Log\Logger())));

return $this->privates['validator.not_compromised_password'] = new \Symfony\Component\Validator\Constraints\NotCompromisedPasswordValidator($a, 'UTF-8', true, NULL);
